### PR TITLE
[Feature]: New 'tertiary' variant

### PR DESCRIPTION
**Pull request by fajar-apri-alaska**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #204

## Summary:

This PR is to provides 'tertiary' option for CTA-hyperlink per ticket #204.

- add the new feature example to apiExamples
- also update generateDocs script to fix the issue in macOS based machines

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Generator script update

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by fajar-apri-alaska** [comment]
Hi team,
News are, the key token values are a bit different with the one from blueprints (still using --auro instead of --ds).
I tried to map out per the latest ones. But I'm still not sure since some value are a bit off by it looks after it is rendered on the browser. So if anything is a miss from my side, please tell me so.
Thanks.

Figma/Blueprint links:
https://github.com/AlaskaAirlines/auro-hyperlink/issues/204#issuecomment-2061749224

**Comment by blackfalcon** [comment]
> Hi team, News are, the key token values are a bit different with the one from blueprints (still using --auro instead of --ds). I tried to map out per the latest ones. But I'm still not sure since some value are a bit off by it looks after it is rendered on the browser. So if anything is a miss from my side, please tell me so. Thanks.
> 
> Figma/Blueprint links: [#204 (comment)](https://github.com/AlaskaAirlines/auro-hyperlink/issues/204#issuecomment-2061749224)

@leeejune please take note that the Figma blueprint is still referencing `--auro-` versus `--ds`.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: scripts/generateDocs.mjs [hunk 1]
```diff
@@ -2,8 +2,9 @@ import path from 'path';
 import markdownMagic from 'markdown-magic';
 import fs from 'fs';
 import https from 'https';
+import { fileURLToPath } from 'url';
 
-const __dirname = new URL('.', import.meta.url).pathname;
+const __dirname = fileURLToPath(new URL('.', import.meta.url));
```
[the remark this answers is not in the document]
Comment (fajar-apri-alaska) [reply scripts/generateDocs.mjs:7]: Hi @blackfalcon,
this is related to last concern in our discussion by chat.

So, when I used the old script: new URL('.', import.meta.url).pathname, it returns the encoded path 
 
`/Users/fajar_a/Git%20Project/AlaskaAirlines/auro-flight/scripts/`
 
that doesn't work on my machine, and will return like below example from my machine.
image
*screenshot taken from auro-flight
 
Meanwhile using fileURLToPath(new URL('.', import.meta.url)), would return
 
`/Users/fajar_a/Git Project/AlaskaAirlines/auro-flight/scripts/`
 
it now works, and render the documentation as expected.

--- 

Should I just do the script on my end only then? I can excludes the change to the script for this commit.

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**